ui/webui/src/apis/boss.js

```diff
@@ -43,12 +55,10 @@ export const getSteps = ({ address, task }) => {
 };
 
 /**
- * @param {string} address      Anaconda bus address
- *
  * @returns {Promise}           Resolves a list of tasks
  */
-export const installWithTasks = ({ address }) => {
-    return bossClient({ address }).call(
+export const installWithTasks = () => {
+    return new BossClient().client.call(
```

Comment:
So this makes use of the singleton ? Interesting. :)

Reply:
Indeed. so now only the first call will result in an new boss interface dbus client creation, etc.

Reply:
Cool! In this case its mainly for performance/efficiency but I'm sure we will hit cases where we will need to make sure something is there once and only once & this will be very handy for that as well. :)